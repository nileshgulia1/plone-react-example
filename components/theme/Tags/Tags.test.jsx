import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';

import Tags from './Tags';

const mockStore = configureStore();

describe('Tags', () => {
  it('renders without tags', () => {
    const store = mockStore({
      intl: {
        locale: 'en',
        messages: {},
      },
    });
    const component = renderer.create(
      <Provider store={store}>
        <Tags />
      </Provider>,
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });

  it('renders with tags', () => {
    const store = mockStore({
      intl: {
        locale: 'en',
        messages: {},
      },
    });
    const component = renderer.create(
      <Provider store={store}>
        <Tags tags={['Tag 1', 'Tag 2']} />
      </Provider>,
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
