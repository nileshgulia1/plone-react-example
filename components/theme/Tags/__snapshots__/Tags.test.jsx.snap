// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags renders with tags 1`] = `
<div>
  <a
    className="ui label"
    onClick={[Function]}
    style={Object {}}
  >
    Tag 1
  </a>
  <a
    className="ui label"
    onClick={[Function]}
    style={Object {}}
  >
    Tag 2
  </a>
</div>
`;

exports[`Tags renders without tags 1`] = `<span />`;
